src/components/01-atoms/Tooltip/index.js

```diff
@@ -30,6 +32,7 @@ Tooltip.propTypes = {
   start: PropTypes.shape({}).isRequired,
   end: PropTypes.shape({}).isRequired,
   roomName: PropTypes.string.isRequired,
+  user: USER_SHAPE,
```

Comment:
This is a great approach. We should follow this pattern elsewhere. There are so many repeated proptypes in bookit-web.